rosbag2_test_common/package.xml

```diff
@@ -16,9 +16,11 @@
 
   <build_depend>rclcpp</build_depend>
   <build_depend>rcutils</build_depend>
+  <build_depend>test_msgs</build_depend>
 
   <exec_depend>rclcpp</exec_depend>
   <exec_depend>rcutils</exec_depend>
+  <exec_depend>test_msgs</exec_depend>
```

Comment:
good point. Looks like all of these `<exec_depend>` could be moved to `<test_depend>`

Reply:
or, well no, since these are not used for testing `rosbag2_test_common` itself, so they should probably be kept as `<build_depend>`/`<exec_depend>`.

But packages using `rosbag2_test_common` for tests should use it as a `<test_depend>`.

Reply:
@fujitatomoya @christophebedard Yes. The last comment is correct. We don't have tests in `rosbag2_test_common` package. However, we are using it as a helper package in other Rosbag2 packages in tests.